src/agent0/hyperfuzz/system_fuzz/invariant_checks.py

```diff
@@ -318,15 +318,23 @@ def _check_negative_interest(interface: HyperdriveReadInterface, pool_state: Poo
         # Different error messages and log levels if the pool is paused
         if interface.get_pool_is_paused():
             exception_message = (
-                "Negative interest detected on paused pool. "
+                "Negative interest detected between block "
+                f"{previous_pool_state.block_number}"
```

Comment:
Can you add a timestamp here?

Reply:
I figured we could look up the timestamp online. Do you want timestamp AND block number or instead of? I was worried that both would clog up the discord notif.

Reply:
I guess that's true, but I'd rather have the information there, shouldn't be too much more info